### PR TITLE
meetup: Clarify description

### PR DESCRIPTION
**Pull request by Insti**
Clarify the problem description for meetup.

This PR is made up of several commits:

* Enumerate valid descriptors.
* Lowercase '*teenth' descriptors.
* Add description of what to do if there is no date.
* Note that all descriptions will be valid.
* Make descriptions consistent.
* Reformat text to fit in 80 columns. 

The goal here is to make the description less ambiguous by explicitly stating the expected date specifier strings and what to do if there is no matching date.

A consistent format for the input date specifications and only testing against valid input strings helps ensure that this remains a problem about date parsing rather than arbitrary string parsing.

The final commits reformat the text to 80 columns for consistency with other exercise descriptions.

### CODE REVIEW

File: exercises/meetup/description.md [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 
-Given examples of a meetup dates, each containing a month, day, year, and descriptor
-(first, second, teenth, etc), calculate the date of the actual meetup.
-For example, if given "First Monday of January 2017", the correct meetup date is 2017/1/2
+Given examples of a meetup dates, each containing a month, day, year, and
+descriptor calculate the date of the actual meetup.  For example, if given 
+"The first Monday of January 2017", the correct meetup date is 2017/1/2
+
+All descriptions will be valid, but you will need to return an error if there
+is no matching date. For example, "The fifth Wednesday of October 2017"
```
Comment (ErikSchierboom) [inline exercises/meetup/description.md]: It feels like the last line is missing something at the end. Maybe add " has no matching date"?

File: exercises/meetup/description.md [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
-Given examples of a meetup dates, each containing a month, day, year, and descriptor
-(first, second, teenth, etc), calculate the date of the actual meetup.
-For example, if given "First Monday of January 2017", the correct meetup date is 2017/1/2
+Given examples of a meetup dates, each containing a month, day, year, and
+descriptor calculate the date of the actual meetup.  For example, if given 
+"The first Monday of January 2017", the correct meetup date is 2017/1/2
+
+All descriptions will be valid, but you will need to return an error if there
+is no matching date. For example, October 2017 has five Tuesdays but only four
+Wednesdays. There is no date that matches "The fifth Wednesday of October 2017"
```
Comment (Insti) [inline exercises/meetup/description.md]: Should we even be testing with invalid dates?
We're not testing other invalid inputs.

I suggest removing the last section completely and leaving error cases undefined by the canonical data.

**Comment by Insti** [comment]
https://github.com/exercism/problem-specifications/pull/982/commits/e5dea5b617726536654218ef73c5d008becda6a1 - Clarifies the invalid date handling description.

https://github.com/exercism/problem-specifications/pull/982/commits/2b4cfb66dc6fcb9b45e9237b865630cd5918ca3a - Leaves invalid date handling undefined.

File: exercises/meetup/description.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+that end in '-teenth'. Therefore, one is guaranteed that each day of the week
 (Monday, Tuesday, ...) will have exactly one date that is named with '-teenth'
 in every month.
 
-Given examples of a meetup dates, each containing a month, day, year, and descriptor
-(first, second, teenth, etc), calculate the date of the actual meetup.
-For example, if given "First Monday of January 2017", the correct meetup date is 2017/1/2
+Given examples of a meetup dates, each containing a month, day, year, and
+descriptor calculate the date of the actual meetup.  For example, if given 
+"The first Monday of January 2017", the correct meetup date is 2017/1/2
```
Comment (ErikSchierboom) [inline exercises/meetup/description.md]: I'm missing a dot (.) at the end of the line, or was that on purpose?.

Comment (ErikSchierboom) [reply exercises/meetup/description.md]: @Insti small bump for my comment.

Comment (Insti) [reply exercises/meetup/description.md]: Thanks, I needed the bump. 👍

**Comment by ErikSchierboom** [comment]
Great improvement!

**Comment by ErikSchierboom** [comment]
@Insti Would you like to keep the various commits or should I do a squash?

**Comment by Insti** [comment]
Personally, I'd keep them, since they are all atomic actions and I prefer to be able to see all the change steps in the git history. 
Other people like squashing for some reason, so do whatever you think is best.

**Comment by ErikSchierboom** [comment]
Merged! Thanks a lot @Insti. :tada:

**Pull request merged**